molecule_doc.py

```diff
@@ -10,7 +10,7 @@
 import os
 
 # Set path to molecule scenarios
-path = 'roles/confluent.test/molecule/'
```

Comment:
You could have made a variable with a single change.

Reply:
There is a var, I just forgot to use it.  I have corrected this now. :D